README.md

```diff
@@ -10,6 +10,14 @@ To contribute, simply make a pull request to this repository with your package c
 We run continuous integration on this repository to test builds of a large number of
 Spack packages.
 
+If you want to test your package changes locally before submitting a pull request,
+simply change Spack's default package repo from the default cache location to the full
+path to your local git clone:
+```
+spack repo set --destination /path/to/local/spack-packages builtin
```

Comment:
I fully acknowledge this is bikeshedding, feel free to ignore.

Do we want suggest that folks re-map their `builtin` repo to their packages checkout simply for the purposes of a one off test? The instructions for how to do this is general are very valuable, it's the first thing I did post repo split, but within the framing of "quickly test this" I don't know that we want to suggest a user overwrite their builtin repo with their specific checkout of the packages repo and then be responsible for maintaining that, particularly if they're not expecting this to modify their primary repository. To us it's pretty obvious since you're specifying `builtin` but that does require some degree of familiarity that might trip new users.

Reply:
@johnwparent how about propose that in a follow-on?